db/schema.yml

```diff
@@ -6154,6 +6154,7 @@ physical_servers:
 - serial_number
 - field_replaceable_unit
 - raw_power_state
+- vendor
```

Comment:
Hi, @bdunne .
`vendor` is a property called by all generic quadicon_helper into manageiq-ui-classic.
All entities that need render some image on interface use this `vendor` property.